docs/contributing/docs.rst

```diff
@@ -50,18 +50,18 @@ __ http://docutils.sourceforge.net/
 
 To build the documentation locally, install Sphinx:
 
-.. console::
+.. code-block:: console
 
      $ pip install Sphinx
 
 Then from the ``docs`` directory, build the HTML:
 
-.. console::
+.. code-block:: console
 
      $ make html
 
-To get started contributing, you'll want to read the :ref:`reStructuredText
-reference <sphinx:rst-index>`.
```

Comment:
What kind of reference was this meant to be? Something that goes straight to the sphinx documentation?

Reply:
Yes, it is! I'm not sure why this isn't working. I copied this from the djago docs.

Reply:
Maybe it requires some kind of setup in `conf.py`? Can you link me to the source of the django docs, then I could investigate.

Reply:
Fixed.